thrift/src/main/thrift/atoms/media.thrift

```diff
@@ -0,0 +1,27 @@
+namespace * contentatom.media
+namespace java com.gu.contentatom.thrift.atom.media
+#@namespace scala com.gu.contentatom.thrift.atom.media
+
+typedef i64 Version
+
+enum AssetType {
+  YOUTUBE,
+  FACEBOOK,
+  PODCAST,
+  DAILYMOTION,
+  MAINSTREAM,
```

Comment:
@fredex42 is the expert, but it is some kind of service that syndicates our video to other sites. We publish our videos there.